src/udev/udev-builtin-blkid.c

```diff
@@ -244,13 +244,14 @@ static int builtin_blkid(struct udev_device *dev, int argc, char *argv[], bool t
         for (;;) {
                 int option;
 
-                option = getopt_long(argc, argv, "oR", options, NULL);
+                option = getopt_long(argc, argv, "o::R", options, NULL);
                 if (option == -1)
                         break;
 
                 switch (option) {
                 case 'o':
-                        offset = strtoull(optarg, NULL, 0);
+                        if (optarg)
+                                offset = strtoull(optarg, NULL, 0);
```

Comment:
I don't get this one.

Could you maybe split out each of the changes into a separate commit with a short explanation?

Reply:
No problem, I just force pushed the new commits with some better explanations.

Reply:
I prefer to use `safe_atollu()` here. What do you think?